### PR TITLE
Bug 1129200 - Listen for wheel events for screen reader swipe input.

### PR DESCRIPTION
**Pull request by eeejay**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=044f483e8dc3)

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1b747b8c98a8)

File: apps/gallery/js/frames.js [hunk 1]
```diff
@@ -72,6 +72,8 @@ frames.addEventListener('dbltap', dblTapHandler);
 frames.addEventListener('pan', panHandler);
 frames.addEventListener('swipe', swipeHandler);
 frames.addEventListener('transform', transformHandler);
+frames.addEventListener('wheel', wheelHandler);
+
```
Comment (punamdahiya) [inline apps/gallery/js/frames.js]: nit: extra line added can be removed

**Comment by try-server-hook** [comment]
[eeejay Eitan Isaacson (eeejay)](https://github.com/eeejay) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f92f68b8fa89)

**Pull request merged**